pnpm-lock.yaml

```diff
@@ -11899,6 +11900,103 @@ packages:
       - aws-crt
     dev: true
 
+  /@aws-sdk/client-sso-oidc/3.569.0:
+    resolution: {integrity: sha512-u5DEjNEvRvlKKh1QLCDuQ8GIrx+OFvJFLfhorsp4oCxDylvORs+KfyKKnJAw4wYEEHyxyz9GzHD7p6a8+HLVHw==}
```

Comment:
I thought I asked about aws-sdk use and duplication in my last review, but I don't see a history of that.
Why the duplication? And per comments in a build-tools update PR, do we need them at all?

Reply:
I can't explain the duplication. I use the same process for updating lockfiles every time and I cannot explain why sometimes it produces some changes and other times it doesn't. I just reset it to main again and regenned it - we'll see if that's any better.

The aws-sdk dependencies and peer dependency requirements come from oclif. We don't use any of that functionality, but I don't think it's worth doing weird things to exclude the direct dependencies. For the peer dependencies, I added ignore peer dep entries in the pnpm config for @aws/* (that change wasn't in the iteration you reviewed)